deploy-with-init-params.md

```diff
@@ -13,15 +13,12 @@ You have completed [this tutorial](smart-contract-deployment-in-forgae.md) that
 We would need a local node to compile and deploy on. The easiest option is to spawn one from forgae. Follow the below steps to create a project, initialize the project, and test the project using forgae on your CLI:
 
 ```
-Step 1: $ mkdir forgae-init
-Step 2: $ cd forgae-init
-Step 3: $ forgae init
-Step 4: $ forgae node or docker-compose for docker toolbox users
 mkdir forgae-init
 cd forgae-init
 forgae init
 forgae node
 ```
+Note: For docker toolbox users, you may need to run `docker-compose` instead of `forgae node`
```

Comment:
in the current version of forgae this will not start the compiler

Reply:
@GoodnessEzeokafor do you mind elaborating what you meant here with running `docker-compose`?

Reply:
i'm using windows 8.1,forgae node didn't start the nodes and the compiler, `docker-compose up` did

Reply:
C:\Users\goote_000\Documents\smart_contracts\voting_app>docker-compose up
Starting voting_app_node2_1 ... done
Starting voting_app_proxy_1 ... done
Starting voting_app_node3_1 ... done
Starting voting_app_node1_1 ... done
Attaching to voting_app_node1_1, voting_app_node3_1, voting_app_node2_1, voting
app_proxy_1

Reply:
In the current version of forgae this will not start the local compiler

Reply:
how do i update mine????

Reply:
and i haven't gotten my bounty for the todo app

Reply:
Ah ok, so maybe this command should be removed from the Tutorial so that it's up-to-date with the latest version of forgae.

How did you install forgae in the beginning? You might need to open an issue in the forgae repository if you need help with upgrading.